app/workers/scheduler/ip_spamlist_urls_scheduler.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class Scheduler::IpSpamlistUrlsScheduler
+  include Sidekiq::Worker
+
+  sidekiq_options retry: 0
+
+  def perform
+    grab_ip_blocks_addresses! if ENV['SCHEDULED_IPBLOCK_URLS'].present?
```

Comment:
Minor style thing, but -- I'd probably just have either one top level thing (`process_ip_urls!` or something) or two (one for limit, one for block), and then have those methods wrap up the "if env var present" loop)

Reply:
I'm not a programmer but my intention is purely readability.

The first function pulls the lists from the array of 1 or more URLS and appends those ips to be processed.
the second function processes the ips.

I don't understand ruby enough to fix this... I hacked my way here and I don't remember how